### PR TITLE
Fix table header cell tags

### PR DESCRIPTION
**Pull request by MattieBelt**
**Description of what you did:**

Changed the cells in the table header from `td` -> `th` as should be used in `thead`.

This allows correct usage of the styled component `Table` with a custom generated table.

### CODE REVIEW

File: packages/buffetjs-styles/src/components/Table/index.js [hunk 1]
```diff
@@ -78,6 +79,7 @@ const Table = styled.div`
       line-height: 0.1rem;
       font-weight: ${sizes.fontWeight.bold};
       text-transform: capitalize;
+      th,
       td {
```
Comment (MattieBelt) [inline packages/buffetjs-styles/src/components/Table/index.js:83]:
```suggestion
      th {
```

Question: Can `td` be removed here, or does this break custom tables in Strapi?

**Comment by MattieBelt** [comment]
@HichamELBSI any update/review?

**Comment by HichamELBSI** [comment]
Hello @MattieBelt, I will review this one today.

File: packages/buffetjs-styles/src/components/Table/index.js (hunk 1 above)
Comment (HichamELBSI) [reply packages/buffetjs-styles/src/components/Table/index.js:83]: You can remove it.

File: packages/buffetjs-core/src/components/TableHeader/index.js [hunk 2]
```diff
@@ -12,11 +12,11 @@ import Icon from '../Icon';
 
 /* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
 function TableHeader({
+  addCell,
```
Comment (HichamELBSI) [inline packages/buffetjs-core/src/components/TableHeader/index.js:15]: Can you keep the verb "shouldAddCell" at the beginning of the variable name, please. This is a naming rule for booleans props.

**Comment by MattieBelt** [comment]
> You can remove it.

:+1: 
> Can you keep the verb "should..." at the beginning of the variable name, please. This is a naming rule for booleans props.

Sure

**Comment by HichamELBSI** [comment]
Thank you for the update @MattieBelt. Can you just update the branch please so we can merge.

**Comment by MattieBelt** [comment]
@HichamELBSI done

**Pull request merged**